rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -558,6 +567,7 @@ pub struct IdRange {
     pub range: Option<(LocalId, u64)>,
 }
 
+#[cfg(feature = "uuid-generation")]
```

Comment:
Long-term, we should only test the few methods enabled with this feature flag with this kind of config.